site/content/docs/4.3/components/dropdowns.md

```diff
@@ -64,63 +64,63 @@ The best part is you can do this with any button variant, too:
 <div class="bd-example">
   <div class="btn-group">
     <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Primary</button>
-    <div class="dropdown-menu">
-      <a class="dropdown-item" href="#">Action</a>
-      <a class="dropdown-item" href="#">Another action</a>
-      <a class="dropdown-item" href="#">Something else here</a>
-      <div class="dropdown-divider"></div>
-      <a class="dropdown-item" href="#">Separated link</a>
-    </div>
+    <ul class="dropdown-menu">
+      <li><a class="dropdown-item" href="#">Action</a></li>
+      <li><a class="dropdown-item" href="#">Another action</a></li>
+      <li><a class="dropdown-item" href="#">Something else here</a></li>
+      <li class="dropdown-divider"></li>
```

Comment:
Not sure if a divider is actually an "item", which carries value.

Reply:
While I agree a divider may not carry value the content model for `ul` elements requires direct children to be exclusively `li` elements

https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element

> Content model:
> Zero or more li and script-supporting elements.